marlowe-playground-client/src/Marlowe/Holes.purs

```diff
@@ -215,36 +217,24 @@ instance semigroupHoles :: Semigroup Holes where
 instance monoidHoles :: Monoid Holes where
```

Comment:
Small thing, but I _think_ if you make `Holes` a newtype, you can do `derive newtype instance monoidHoles :: Monoid Holes`, which is neat.

(Also you'll get some free performance by making it a newtype.)